### PR TITLE
Point2/4 constructor Initialize all elements to the same value

### PR DESCRIPTION
**Pull request by rextimmy**
Just fixed up Point2/4 so you can initialize all elements to the same value like in mPoint3I/F/D

### CODE REVIEW

**Comment by rextimmy** [comment]
For consistency I really should have added a Point4D class to this.

File: Engine/source/math/mPoint2.h [hunk 1]
```diff
@@ -285,6 +288,13 @@ inline Point2I::Point2I(S32 _x, S32 _y)
 }
 
 
+inline Point2I::Point2I(S32 xy)
+   : x(xy), y(xy)
+{
+   //
```
Comment (dottools) [inline Engine/source/math/mPoint2.h]: Why do these empty comments exist? Why not have it an entirely empty function like what you did in mPoint4.h? I find it cleaner that way.

Comment (rextimmy) [inline Engine/source/math/mPoint2.h]: Yeah cleaner without them but just matching what is already there ;)  https://github.com/rextimmy/Torque3D/blob/mPointX_constructors/Engine/source/math/mPoint2.h#L271-L288

Comment (dottools) [inline Engine/source/math/mPoint2.h]: That's so ugly and pointless. :(

Comment (rextimmy) [inline Engine/source/math/mPoint2.h]: Yeah it's the same story in mPoint3.h too

**Pull request closed without merging**